unstable/wlr-virtual-pointer-unstable-v1.xml

```diff
@@ -124,5 +131,19 @@
       <arg name="seat" type="object" interface="wl_seat" allow-null="true"/>
       <arg name="id" type="new_id" interface="zwlr_virtual_pointer_v1"/>
     </request>
+    <request name="destroy" type="destructor" since="1">
+      <description summary="destroy the virtual pointer manager"/>
+    </request>
+
+    <request name="create_virtual_pointer2" since="2">
```

Comment:
A convention is to add a `<!-- Version 2 additions -->` comment before the new requests/events so that we make sure not to add anything out-of-order.